__tests__/apis.test.js

```diff
@@ -14,19 +15,55 @@ afterAll(() => {
 });
 
 describe('1. GET /api/categories', () => {
-    test('status: 200, responds with an array of category objects including slugs and descriptions', () => {
-      return request(app)
-        .get('/api/categories')
-        .expect(200)
-        .then(({ body: {categories} }) => {
-          categories.forEach((category) => {
-            expect(category).toEqual(
-              expect.objectContaining({
-                slug: expect.any(String),
-                description: expect.any(String),
-              })
-            );
-          });
+  test('status: 200, responds with an array of category objects including slugs and descriptions', () => {
+    return request(app)
+      .get('/api/categories')
+      .expect(200)
+      .then(({ body: {categories} }) => {
+        categories.forEach((category) => {
```

Comment:
You will need to assert the length of the array here - otherwise the forEach will never be executed